lib/lltz_ir/dsl.ml

```diff
@@ -0,0 +1,183 @@
+(* dsl.ml *)
+open Grace
+module LE = Expr
+module LT = Type
+module LP = Primitive
+
+module T = struct
```

Comment:
"T" is not a good module name I think. What does it stand for?

Reply:
`module T` is often used when the base type of the current module is used in a functor e.g. 
```ocaml
module T = struct 
  type t = { foo: int } [@@deriving sexp, equal, compare]
end
include T
include Comparable.Make (T)
```

However, in this case it is a bit weird since we have no functors that rely on these types.

Reply:
what is the prefered way of doing it here then? renaming T, not using module for it, or something else?